src/components/Dashboard/BookFilter.js

```diff
@@ -1,11 +1,56 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 
-function BookFilter() {
+import './BookFilter.css';
+import trianglePng from '../../assets/triangle.png';
+
+const showOptions = () => document.getElementById('dropdown-content').classList.toggle('show');
```

Comment:
Handle adding or removing classes with react, i.e.:

```js
<div className=`dropdown-content ${this.state.dropdownOpen ? 'show' : '' }`>
```

Direct manipulation on the dom outside the react component lifecycle is a bad pattern